packages/inventory-insights/src/Insights.js

```diff
@@ -19,17 +19,17 @@ import {
     ToolbarGroup,
     ToolbarItem
 } from '@patternfly/react-core';
-import { CheckIcon, ExternalLinkAltIcon, SearchIcon, TimesCircleIcon, PficonSatelliteIcon } from '@patternfly/react-icons';
+import { CheckIcon, ExternalLinkAltIcon, SearchIcon, TimesCircleIcon, PficonSatelliteIcon, AnsibeTowerIcon } from '@patternfly/react-icons';
 import { cellWidth, sortable, SortByDirection, Table, TableBody, TableHeader } from '@patternfly/react-table';
 import { flatten, sortBy } from 'lodash';
-import { global_success_color_200 } from '@patternfly/react-tokens';
+import { global_success_color_200, global_BackgroundColor_100 } from '@patternfly/react-tokens';
```

Comment:
We should create styles for this, but that is not in scop of this PR so good to merge. Just to remember for future.